### PR TITLE
PPTT-3102 described two new javascript tracking client functions

### PR DESCRIPTION
**Pull request by prk3**

### CODE REVIEW

File: data_collection/web/javascript_tracking_client/api.rst [hunk 1]
```diff
@@ -2309,3 +2309,57 @@ Miscellaneous
             .. code-block:: javascript
 
                 jstc.trackError(new Error("Uncaught SyntaxError"));
+
+.. function:: getTrackingSource()
+
+    Returns tracking source data identifying the library and passed in ``ts_n`` and ``ts_v`` tracking parameters.
+    The default tracking source can be overwritten using :ref:`setTrackingSource<jtc-api-setTrackingSource>` function.
```
Comment (RomanSek) [inline data_collection/web/javascript_tracking_client/api.rst:2316]:
```suggestion
    Returns tracking source name and version that identifies the library sending request for debugging purposes.
    The default tracking source is ``jstc`` and can be overwritten using :ref:`setTrackingSource<jtc-api-setTrackingSource>` function.
```

File: data_collection/web/javascript_tracking_client/api.rst [hunk 2]
```diff
@@ -2309,3 +2309,57 @@ Miscellaneous
...
+                console.log("version: " + nameAndVersion[1]);
+
+.. _jtc-api-setTrackingSource:
+
+.. function:: setTrackingSource(name, version)
+
+    Overwrites the default tracking source.
+
+    :param string name: **Required** Tracking source name, e.g. ``"custom-source"``
+    :param string version: Tracking source version, e.g. ``"1.0.0"``. If skipped, the version will not change.
```
Comment (RomanSek) [inline data_collection/web/javascript_tracking_client/api.rst:2349]:
```suggestion
    :param string version: **Optional** Tracking source version, e.g. ``"1.0.0"``. If skipped, the version will not change.
```

**Review by RomanSek — COMMENTED** [review]
[Preview](https://developers.piwik.pro/en/feature-pptt-3102-source-tracking/data_collection/web/javascript_tracking_client/api.html#getTrackingSource)

File: data_collection/web/javascript_tracking_client/api.rst (hunk 1 above)
Comment (prk3) [reply data_collection/web/javascript_tracking_client/api.rst:2316]: We don't collect source data only for debugging, we might at some point show a report with sources. How about this?
```suggestion
    Returns tracking source name and version that identifies the library sending tracking requests.
    This data will be passed in ``ts_n`` and ``ts_v`` tracking parameters.
    The default tracking source is ``jstc`` and can be overwritten using :ref:`setTrackingSource<jtc-api-setTrackingSource>` function.
```

Comment (RomanSek) [reply data_collection/web/javascript_tracking_client/api.rst:2316]: OK, just drop the specifics about `ts_n`` and ``ts_v`` tracking parameters. We don't mention how data is sent in other parts of documentation.

**Review by RomanSek — APPROVED** [review]

File: data_collection/web/javascript_tracking_client/api.rst (hunk 1 above)
Comment (prk3) [reply data_collection/web/javascript_tracking_client/api.rst:2316]:
```suggestion
    Returns tracking source name and version that identifies the library sending tracking requests.
    The default tracking source is ``jstc`` and can be overwritten using :ref:`setTrackingSource<jtc-api-setTrackingSource>` function.
```

**Pull request merged**